packages/cli/CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## [5.9.0](https://github.com/stoplightio/prism/compare/v5.8.3...v5.9.0) (2024-08-02)
```

Comment:
Is this change log valid? Seems that above link is broken as there is neither 5.8.3, nor 5.9.0 tags in the repo.

Please note that, release please currently generates false change logs, see https://github.com/stoplightio/prism/pull/2558

Reply:
Hi @lukaszm-sb 
This was my PR https://github.com/stoplightio/prism/pull/2577/files, When I merge those PR changes, then this new PR **chore: release master** is created with this changes.